Libraries/StyleSheet/__tests__/processAspectRatio-test.js

```diff
@@ -47,4 +47,18 @@ describe('processAspectRatio', () => {
     expect(() => processAspectRatio('1 / 1 1')).toThrowErrorMatchingSnapshot();
     expect(() => processAspectRatio('auto 1/1')).toThrowErrorMatchingSnapshot();
   });
+
+  it('should not accept non string falsy types', () => {
+    const invalidThings = [undefined, null, false];
```

Comment:
Isn't `null` an allowed value for all properties?

Reply:
It is allowed and ignored silently in this code.

Reply:
Perhaps we could rename this test to `it('should ignone non string falsy types', () => {`

Reply:
Done